xonsh/timings.py

```diff
@@ -46,10 +46,10 @@ def clocku():
             """clocku() -> floating point number
             Return the *USER* CPU time in seconds since the start of the process.
             This is done via a call to resource.getrusage, so it avoids the
-            wraparound problems in time.clock()."""
+            wraparound problems in time.perf_counter()."""
```

Comment:
So, as `clock` is deprecated and won't be used again anyway, we can probably just remove the comment?